palm/plugins/core/templates/plugin/setup.py

```diff
@@ -0,0 +1,36 @@
+from distutils.core import setup
+from pathlib import Path
+import sys
+
+# require version of setuptools that supports find_namespace_packages
+from setuptools import setup
+
+try:
+    from setuptools import find_namespace_packages
+except ImportError:
+    # the user has a downlevel version of setuptools.
+    print('Error: palm requires setuptools v40.1.0 or higher.')
+    print(
+        'Please upgrade setuptools with "pip install --upgrade setuptools" and try again'
+    )
+    sys.exit(1)
+
+# User-friendly description from README.md
+this_directory = Path(__file__).parent
+long_description = Path(this_directory, 'README.md').read_text()
+
+setup(
+    name='palm-{{plugin_name}}',
+    version='0.0.1',
+    description='{{plugin_name}} extension for Palm CLI',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    author='Palmetto - Data & Analytics team',
```

Comment:
nitt: do we want to prompt for this maybe?

Reply:
or like "your-cool-team@something.com"

Reply:
Yeah, good shout, I'll bake this in to the generator